opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -2591,6 +2728,61 @@ namespace Opm {
         }
     }
 
+
+
+
+
+    template<typename TypeTag>
+    void
+    BlackoilWellModel<TypeTag>::
+    assignWellGuideRates(data::Wells& wsrpt) const
+    {
+        for (const auto& well : this->wells_ecl_) {
+            auto xwPos = wsrpt.find(well.name());
+            if (xwPos == wsrpt.end()) { // No well results.  Unexpected.
+                continue;
+            }
+
+            auto& grval = xwPos->second.guide_rates;  grval.clear();
+            grval += this->getGuideRateValues(well);
```

Comment:
Is `+=` intentional here?

Reply:
> Is `+=` intentional here?

Yes.  We `.clear()` the value on the previous line.

Reply:
That is not very visible, please put the clear() on a separate line.

Reply:
I know it is probably equivalent to `=`.  Just wondering if there is extra reasons to use `+=`?

Reply:
I've pushed an update here.  I'd forgotten that [`GuideRateValue`](https://github.com/OPM/opm-common/blob/968aa79468ddef5e901538b8d09d2b46b011b379/opm/output/data/GuideRateValue.hpp#L31) supports direct assignment.